client/src/Components/General/SubmitButton.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const SubmitButton = (text) => {
```

Comment:
What's this for?

Reply:
At the moment, this is a placeholder for a submission button that can be recycled throughout the app. Will delete it though since it falls outside the scope of the purpose of this branch.